kernel/src/lib.rs

```diff
@@ -7,8 +7,10 @@
 //! implenents all these traits and can be used to ease integration work. See
 //! [`DefaultEngine`](engine/default/index.html) for more information.
 //!
-//! A full `rust` example for reading table data using the default engine can be found
-//! [here](https://github.com/delta-incubator/delta-kernel-rs/blob/main/kernel/examples/dump-table/src/main.rs)
+//! A full `rust` example for reading table data using the default engine can be found in the
+//! [inspect-table] example.
```

Comment:
good catch, but the link we actually probably want is https://github.com/delta-incubator/delta-kernel-rs/tree/main/kernel/examples/read-table-single-threaded (and call it `read-table-single-threaded`). Maybe link to https://github.com/delta-incubator/delta-kernel-rs/tree/main/kernel/examples/read-table-multi-threaded as well and just say a more complex multi-threaded example also exists